docs/quickstart.rst

```diff
@@ -51,22 +51,22 @@ This includes arithmetic, :doc:`numpy.ufunc <reference/ufuncs>` operations, or f
 .. code-block:: python
 
    >>> np.sin(s) + s.T * 1
-   <COO: shape=(100, 100, 100), dtype=float64, nnz=189601>
+   <COO: shape=(100, 100, 100), dtype=float64, nnz=189601, fill_value=0.0>
 
-However, operations which convert the sparse array into a dense one will raise exceptions
-For example, the following raises a :obj:`ValueError`.
+However, operations which convert the sparse array into a dense one will usually change the fill
+value instead of raising an error.
```

Comment:
The new sentence here is a little confusing to me. These operations how change the fill value instead of converting a sparse array to a dense array, so they don't "convert the sparse array into a dense one" at all now.

Reply:
Changed the sentence structure a bit.